includes/formatting.php

```diff
@@ -106,7 +106,10 @@ function give_maybe_sanitize_amount( $number, $dp = false, $trim_zeros = false )
 	// Handle thousand separator as '.'
 	// Handle sanitize database values.
 	$number_parts = explode( '.', $number );
-	$is_db_sanitize_val = ( 2 === count( $number_parts ) && ( 6 === strlen( $number_parts[1] ) ) );
+	$is_db_sanitize_val = ( 2 === count( $number_parts ) &&
```

Comment:
@DevinWalker I improved this flag because only DB sanitize value can pass these parameters and after exploding by `.` both values must be numeric type.